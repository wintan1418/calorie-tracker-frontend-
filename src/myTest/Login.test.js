import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import LogIn from '../component/display/login';

it ('renders accurately', ()=> {
  const tree = renderer.create(
    <BrowserRouter>
    <LogIn/>
    </BrowserRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});