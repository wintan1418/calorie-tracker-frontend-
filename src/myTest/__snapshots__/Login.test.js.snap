// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders accurately 1`] = `
<div
  className="authen-page"
>
  <div>
    <i
      className="fas fa-user-circle"
    />
  </div>
  <form
    onSubmit={[Function]}
  >
    <input
      placeholder="Your email"
      required={true}
      type="email"
    />
    <input
      placeholder="Password"
      required={true}
      type="password"
    />
    <button
      type="submit"
    >
      Log in
    </button>
  </form>
  <div
    className="authen-text"
  >
    <div>
      Yet to sign up? click here
    </div>
    <div>
      Go to
      <a
        href="/register"
        onClick={[Function]}
      >
        Sign up page
      </a>
    </div>
  </div>
  <div
    className="calorie-text authen-text"
  />
</div>
`;
